Document/0x03-Overview.md

```diff
@@ -1,75 +1,86 @@
+> some comments are in blocks like this, some are in italics within paragraphs. In-line italics are marked with a "Q-" so you can search for those instances  
+> to do a technical review of my edits, track changes within git or paste the past version and this version of the doc into a diffchecker such as https://www.diffchecker.com/
+
 # Introduction to the OWASP Mobile Security Testing Guide
 
-The OWASP Mobile Security Testing Guide (MSTG) is an extension of the OWASP Testing Project specifically focusing on the security testing of Android and iOS devices.
+The OWASP Mobile Security Testing Guide (MSTG) is an extension of the OWASP Testing Project that performs security testing of Android and iOS mobile devices.
+
+The goal of this project is to help people understand everything they need to know about testing Android and iOS applications. MSTG delivers a complete suite of test cases designed to address the [OWASP Mobile Top 10 security risks](https://www.owasp.org/index.php/Mobile_Top_10_2016-Top_10) and the requirements of the [Mobile Application Security Verification Standard (MASVS)](https://github.com/OWASP/owasp-masvs).
+
+>Do we want to give links to those 3 resources/standards mentioned above (if these links exist)? Do we want to link to the OWASP Testing Project? May also want to link the first instance of OWASP to the main site? Are we avoiding links? What are the ways/forms this book is being published?*
+
+## Why Do We Need a Mobile Application Security Testing Guide?
 
-The goal of this project is to help people understand the what, why, when, where, and how of testing applications on Android and iOS devices. The project delivers a complete suite of test cases designed to address the OWASP Mobile Top 10, the Mobile App Security Checklist and the Mobile Application Security Verification Standard (MASVS).
+New technology always introduces new security risks, and mobile computing is no exception. Security concerns for mobile apps differ from traditional desktop software in some important ways, including portability and novelty. Modern mobile operating systems are arguably more secure than traditional desktop operating systems, but problems can still appear when we don't carefully consider security during mobile app development. Data storage, inter-app communication, proper usage of cryptographic APIs, and secure network communication are only some of these considerations. 
```

Comment:
> including portability and novelty

I don't think "novelty" fits here. First, you already mentioned "New technology" and "traditional desktop software", and I don't think there is a need to underline the novelty again here. However, I also don't think it is an important way the security concerns differ.